### PR TITLE
HOMEWORK-5-2: - done;

### PR DESCRIPTION
**Pull request by jah701**

### CODE REVIEW

File: src/main/java/core/basesyntax/Plane.java [hunk 1]
```diff
@@ -7,5 +7,77 @@
  */
 
 public class Plane {
+    private String model;
+    private String color;
+    private int weight;
+    private int maxSpeed;
+    private int passengerCapacity;
 
+    public Plane(PlaneBuilder planeBuilder) {
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/Plane.java]: [checklist](https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_05.html) check about all requirements

File: src/main/java/core/basesyntax/Plane.java [hunk 2]
```diff
@@ -7,5 +7,77 @@
...
+
+    public static class PlaneBuilder {
+        private String model;
+        private String color;
+        private int weight;
+        private int maxSpeed;
+        private int passengerCapacity;
+
+        public PlaneBuilder() {
+        }
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/Plane.java]: do we need to create this type of constructor or we have it by default?

File: src/main/java/core/basesyntax/Plane.java (hunk 1 above)
Comment (jah701) [reply src/main/java/core/basesyntax/Plane.java]: fixed :)

File: src/main/java/core/basesyntax/Plane.java (hunk 2 above)
Comment (jah701) [reply src/main/java/core/basesyntax/Plane.java]: i suggest well get it by default too :) ty

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**